### DISCUSSION TITLE
Merhaba Benim Ender3 V2 Yardıma ihtiyacım var

### DISCUSSION THREAD

**Discussion by gokhanselen** [Jyers/Marlin#163] [Q&A] [answered]
Merhaba Benim Ender3 V2 4.2.7 Anakart bltouch ve Filament sensörü için en uygun yazılımı nerede bulabilirim yardımınız için teşekkür ederim

**Comment by gokhanselen** [comment]
Hello My Ender3 V2 4.2.7 Where can I find the most suitable software for Motherboard bltouch and Filament sensor thank you for your help

**Comment by Jyers** [comment, marked as answer]
You would want one of the bin files labeled with BLTouch and V427
